src/webAppCreator.ts

```diff
@@ -402,11 +412,23 @@ class AppServicePlanStep extends WebAppCreatorStepBase {
     }
 }
 
-class WebsiteStep extends WebAppCreatorStepBase {
+interface WebsiteStepResources {
+    title: string;    // like "Create Web App"
```

Comment:
The only difference in these strings is the Resource name (Web App). Can't we infer that from the appKind?

Reply:
You could if you want to have English terminology inside of localized strings.

Reply:
'Web App' or 'Function App' are product names - so I don't think they would be localized anyway